model/VocabularyConfig.php

```diff
@@ -108,7 +108,7 @@ public function showChangeList()
      * get the URLs from which the vocabulary data can be downloaded
      * @return array Array with MIME type as key, URL as value
      */
-    public function getDataURLs()
+    public function getDataURLs($lang = null)
```

Comment:
This adds a new parameter `$lang`, but I don't see where this parameter is passed to the function?

Reply:
Ok, I'll remove the option to specify the language for marcxml downloads. Instead, I'll let all available language versions to be available.